app/models/Subscription.scala

```diff
@@ -111,9 +113,9 @@ object Subscription {
   }
 
   def create(subscriberId: Int, topicId: Int, action: String, earliest: Date, latest: Date) = {
-    val created = new Date
-    val updated = created
-    val cancelled = created
+    val created = new Date  // todo: should db defaults should handle this?
+    val updated = created   // todo: should db defaults should handle this?
+    val cancelled = created // todo: null might be better?
```

Comment:
Yea, but these might work differently in the future. The date handling around subscriptions will require attention at some point, so the comment is very useful